src/luascript.cpp

```diff
@@ -2033,6 +2037,7 @@ void LuaScriptInterface::registerFunctions()
 	registerMethod("Game", "getWorldType", LuaScriptInterface::luaGameGetWorldType);
 	registerMethod("Game", "setWorldType", LuaScriptInterface::luaGameSetWorldType);
 
+	registerMethod("Game", "getItemAttributeByName", LuaScriptInterface::luaGameGetItemAttributeByName);
```

Comment:
no idea where functions from `tools.cpp` should go. `getReturnMessage` is in `Game`, so I've gone with it for `stringToItemAttribute` too.